package.json

```diff
@@ -75,6 +75,11 @@
     "chai": "^3.2.0",
     "chai-http": "^1.0.0",
     "css-loader": "^0.26.4",
+    "eslint": "^3.19.0",
```

Comment:
will this give conflict & headache to the people who don't have eslint set up yet?